AnnoyancesFilter/Widgets/sections/widgets.txt

```diff
@@ -306,7 +303,7 @@ eegg.fun#?#.innerframe > div.whiteboard > div > div.indent > div[id^="rss_"]:upw
 ||eegg.fun/js/sougo_list.js
 blog.livedoor.jp###main > div.column-inner > div.column-inner-2 > div[style="height:440px; overflow: auto; text-align:left; "]
 blog.livedoor.jp##.RSS04 + div[style^="width:720px;height:160px"]
-[$path=/\/(?:itsoku|news_aru)/]blog.livedoor.jp##.headline
+[$path=/\/(?:itsoku|news_aru)/]blog.livedoor.jp,burusoku-vip.com##.headline
```

Comment:
Do not merge this, rules with `[$path=/\/(?:itsoku|news_aru)/]` should be kept separate.

Reply:
ok